src/include/ceph_fs.h

```diff
@@ -388,6 +388,8 @@ enum {
 	CEPH_MDS_OP_LOOKUPINO  = 0x00104,
 	CEPH_MDS_OP_LOOKUPNAME = 0x00105,
 	CEPH_MDS_OP_GETVXATTR  = 0x00106,
+	CEPH_MDS_OP_DUMMY = 0x00107,
+	CEPH_MDS_OP_EVICT_CLIENT = 0x00108,
```

Comment:
The `CEPH_MDS_OP_EVICT_CLIENT` seems incorrect here.

IMO you should define a dummy one in `include/ceph_fs.h`, just after `CEPH_SESSION_REQUEST_FLUSH_MDLOG` ?

Reply:
Yeah, we are targeting `client_session` so it should be under this enum